osdk/tools/docker/gen_dockerfile.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+
+# The Dockerfile generator for OSDK.
+
+output_dir="ubuntu"
```

Comment:
This default value is surprising. Usually, the user of a CLI tool would NOT expect the tool to output files under a directory other than the current working directory. Remove the `ubuntu` directory.